### PR TITLE
Add blocks for copying specific version link and latest version link individually

### PR DESCRIPTION
**Pull request by abhi14nexu**
## Fixes

Fixes #[849](https://github.com/creativecommons/vocabulary/issues/849) by @[abhi14nexu](https://www.github.com/abhi14nexu)

## Description

This PR adds the feature of copying a specific version link(`v2020.11.1` in Stylesheets and `1.0.0-beta.6`

 in the Javascript section) and the latest version link individually by keeping both the links in individual blocks.

## Screenshots

Screenshot from 2021-02-18 23-20-39
Screenshot from 2021-02-18 23-21-01
## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the *default* branch of the repository (`main` or `master`).
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

File: packages/vocabulary/.storybook/meta/usage.stories.mdx [hunk 1]
```diff
@@ -74,20 +74,26 @@ mirror our `npm` deploys so choose either one based on your personal preference.
 Just link to the master CSS file with any of these one-line `link` tags within your document `head` tag.
 
 #### unpkg
+####
```
Comment (nimishbongale) [inline packages/vocabulary/.storybook/meta/usage.stories.mdx:77]: Not quite sure what an empty #### does? Is there a better way to space it out?

File: packages/vocabulary/.storybook/meta/usage.stories.mdx [hunk 2]
```diff
@@ -96,20 +102,26 @@ Just link to the master CSS file with any of these one-line `link` tags within y
 Source the bundled JS file with any of these one-link `script` tags within your document `head` tag.
 
 #### unpkg
+####
 
+###### 1.0.0-beta.6 Version
```
Comment (nimishbongale) [inline packages/vocabulary/.storybook/meta/usage.stories.mdx:107]: Kindly change all the beta links to more recent ones.

File: packages/vocabulary/.storybook/meta/usage.stories.mdx [hunk 3]
```diff
@@ -74,20 +74,26 @@ mirror our `npm` deploys so choose either one based on your personal preference.
...
 #### unpkg
+####
 
+###### 2020.11.1 Version
 ```html
-<!-- If you have a specific version in mind, use -->
 <link rel="stylesheet" href="https://unpkg.com/@creativecommons/vocabulary@2020.11.1/css/vocabulary.css">
-<!-- or, if you prefer the latest version, use -->
+```
+###### Latest Version
```
Comment (nimishbongale) [inline packages/vocabulary/.storybook/meta/usage.stories.mdx:83]: A line break in between 82 & 83 might make it a little better?

**Review by nimishbongale — CHANGES_REQUESTED** [review]
Good start @abhi14nexu!

A few suggested changes, please do look into them. 👍

File: packages/vocabulary/.storybook/meta/usage.stories.mdx (hunk 2 above)
Comment (abhi14nexu) [reply packages/vocabulary/.storybook/meta/usage.stories.mdx:107]: @nimishbongale Can you please tell me which links are the recent ones as I am unable to find them?

**Review by zackkrida — CHANGES_REQUESTED** [review]
This looks good once @nimishbongale's changes are made. This is a very nice addition, thank you!

File: packages/vocabulary/.storybook/meta/usage.stories.mdx (hunk 2 above)
Comment (nimishbongale) [reply packages/vocabulary/.storybook/meta/usage.stories.mdx:107]: Sample unpkg link (v2020.11.3):

- https://unpkg.com/@creativecommons/vocabulary@2020.11.3/

Sample jsdelivr link (v2020.11.3):

- https://cdn.jsdelivr.net/npm/@creativecommons/vocabulary@2020.11.3/

**Comment by brylie** [comment]
@abhi14nexu, thank you so much for the contribution.

I'm going to close this pull request without merging the changes since it is stale and there are unresolved review comments. Please feel free to re-open the pull request if you have the time to continue the task :smiley:

**Pull request closed without merging**